build/build.ps1

```diff
@@ -34,6 +35,7 @@ function Print-Usage() {
     Write-Host "  -rebuild                Rebuild solution"
     Write-Host "  -deploy                 Deploy built VSIXes"
     Write-Host "  -test                   Run all unit tests in the solution"
+	Write-Host "  -perf                   Run all performance tests in the solution"
```

Comment:
Use spaces instead of tabs for indentation

Reply:
Done...